__tests__/app.test.js

```diff
@@ -84,42 +84,32 @@ describe("GET /api/articles/:articleId", () => {
   });
 });
 
-describe.only("GET /api/articles", () => {
+describe("GET /api/articles", () => {
   test("Should return status 200 and an array including all articles in the right format", () => {
     return request(app)
       .get("/api/articles")
-      .expect(200)
+      .then((res) => {
+        if (res.body.length !== 0) {
+          expect(200);
+        } else {
+          expect(404);
+        }
```

Comment:
This is interesting, a more simplified way of checking for this may be to simply have an expect with the condition in your if statement, which should be true as we should always expect the length of this array to be greater than 0.